CSLabs.Api/Startup.cs

```diff
@@ -135,6 +154,10 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
             app.UseRouting();
 //            app.UseHttpsRedirection();
             app.UseAuthentication();
+            
+            app.UseHangfireDashboard();
```

Comment:
We will want to discuss when we have this exposed.
Should it be exposed in prod and staging and dev or only staging and dev?
In either case we will need to set a conditional around it to activate or not.

Reply:
I agree - add a flag in the appsettings to enable this.

Reply:
is this dashboard read only?